druid-shell/src/backend/windows/window.rs

```diff
@@ -1382,15 +1380,25 @@ impl WindowBuilder {
             let mut dwStyle = WS_OVERLAPPEDWINDOW;
             let mut dwExStyle: DWORD = 0;
             let mut focusable = true;
-            let mut parent: Option<crate::WindowHandle> = None;
             if let Some(level) = self.level {
                 match level {
                     WindowLevel::AppWindow => (),
-                    WindowLevel::Tooltip(p) => {
+                    WindowLevel::Tooltip(parent_window_handle) => {
                         dwStyle = WS_POPUP;
                         dwExStyle = WS_EX_NOACTIVATE | WS_EX_TOOLWINDOW;
                         focusable = false;
-                        parent = Some(p)
+                        if let Some(point_in_window_coord) = self.position {
+                            let screen_point = parent_window_handle.get_position()
+                                + point_in_window_coord.to_vec2();
+                            let scaled_tooltip_point = WindowBuilder::scale_sub_window_position(
+                                Some(screen_point),
+                                parent_window_handle.get_scale(),
+                            );
+                            pos_x = scaled_tooltip_point.x as i32;
+                            pos_y = scaled_tooltip_point.y as i32;
+                        } else {
+                            warn!("No position provided for tooltip!");
```

Comment:
This warning sounds reasonable to me, but maybe there should be something in the `WindowBuilder` API docs about tooltips expecting to have a position?

Reply:
Yes there should be. I have added sub window specific doc to `WindowBuilder::set_position()`.